data/human/variants.txt

```diff
@@ -1161,8 +1161,8 @@ ship "Fury" "Fury (Missile)"
 	outfits
 		"Javelin Pod"
 		"Javelin" 200
-		"Meteor Missile Launcher" 2
-		"Meteor Missile" 75
+		"Meteor Missile Pod" 2
+		"Meteor Missile" 14
 		"Meteor Missile Box"
```

Comment:
This missile box is now unused. Worth removing?

Reply:
Yes, removed. Thanks!